.travis.yml

```diff
@@ -4,7 +4,6 @@ node_js:
   - "6"
 before_script:
```

Comment:
small nit: now that we rely on `npm`, we don't need to globally install `grunt` anymore. Can you remove this line and the next?